### ISSUE TITLE
ViewChange is failing

### ISSUE THREAD

**Issue by ineiti** [dedis/cothority#1583]
When waking up from sleep, the viewchange is failing:

```
2 18/15/11 11:02:19.310714300: (   byzcoin.(*Service).updateTrieCallback: 967) - tls://192.168.0.42:7770 Updating transactions for dc828501cc1a56a942c375d4506dbac018112b9ab361e0baa081b9aec29911bc on index 36
2 18/15/11 11:02:19.584231100: (   byzcoin.(*Service).updateTrieCallback: 1018) - tls://192.168.0.42:7770 updating heartbeat monitor for dc828501cc1a56a942c375d4506dbac018112b9ab361e0baa081b9aec29911bc with window 50s
2 18/15/11 11:02:53.776988300: (                onet.wsHandler.ServeHTTP: 199) - ws request from 172.17.0.1:55194: Skipchain/GetUpdateChain
2 18/15/11 11:02:53.788322600: (          onet.wsHandler.ServeHTTP.func1: 163) - ws close 172.17.0.1:55194 n 1 rx 34 tx 132315
2 18/15/11 11:02:58.862550300: (                onet.wsHandler.ServeHTTP: 199) - ws request from 172.17.0.1:55196: Skipchain/GetUpdateChain
2 18/15/11 11:02:58.873057600: (          onet.wsHandler.ServeHTTP.func1: 163) - ws close 172.17.0.1:55196 n 1 rx 34 tx 132315
2 18/15/11 11:03:20.745524300: (network.NewTLSListenerWithListenAddr.func1: 249) - Got new connection request from: 172.17.0.1:35414
2 18/15/11 11:03:20.764741300: (                  network.(*Router).Send: 211) - tls://[::]:7770 Couldn't send to tls://192.168.0.42:7774 : Connection Closed trying again
2 18/15/11 11:03:20.764874800: (                      network.NewTLSConn: 407) - NewTLSConn to: tls://192.168.0.42:7774
2 18/15/11 11:04:01.471334300: (                onet.wsHandler.ServeHTTP: 199) - ws request from 172.17.0.1:55236: Skipchain/GetUpdateChain
2 18/15/11 11:04:01.479217400: (          onet.wsHandler.ServeHTTP.func1: 163) - ws close 172.17.0.1:55236 n 1 rx 34 tx 132315
panic: view is not equal

goroutine 68 [running]:
github.com/dedis/cothority/byzcoin/viewchange.(*stateLog).add(0xc420453390, 0xc420149bc0, 0x20, 0x20, 0xc420148e60, 0x20, 0x20, 0x1, 0xad59ba326ce1d6f7, 0x6e402922b88ce49b, ...)
	/Users/ligasser/src/github.com/dedis/cothority/byzcoin/viewchange/viewchange.go:369 +0x279
github.com/dedis/cothority/byzcoin/viewchange.(*stateLogs).add(0xc420453540, 0xc420149bc0, 0x20, 0x20, 0xc420148e60, 0x20, 0x20, 0x1, 0xad59ba326ce1d6f7, 0x6e402922b88ce49b, ...)
	/Users/ligasser/src/github.com/dedis/cothority/byzcoin/viewchange/viewchange.go:406 +0xea
github.com/dedis/cothority/byzcoin/viewchange.(*Controller).Start(0xc420086870, 0xad59ba326ce1d6f7, 0x6e402922b88ce49b, 0xc4200216c0, 0x20, 0x20, 0xba43b7400, 0x1)
	/Users/ligasser/src/github.com/dedis/cothority/byzcoin/viewchange/viewchange.go:145 +0xd6a
created by github.com/dedis/cothority/byzcoin.(*viewChangeManager).start
	/Users/ligasser/src/github.com/dedis/cothority/byzcoin/viewchange.go:45 +0x100
make: *** [run-conode] Error 2
```

This might be just an artifact fromt he fact that the node is not active and then suddenly wakes up and times out - anyway, if the panic can go away and only have a `log.Error`, that might be better.

**Comment by kc1212** [comment]
This issue was automatically closed with 9373eca, but the panic was there because it should never happen. Further investigation is needed to find the root cause.

**Comment by ineiti** [comment]
Also see #1691
